visualizations/load-aware/plotting/pod_times.py

```diff
@@ -17,6 +17,8 @@ def register():
 def generatePodTimes(entry):
     data = []
     for p in entry.results.pods_info:
+        # We will need to generate different plots for each workload type in the future
+        if "sleep" in p.pod_name: continue
```

Comment:
🤔 
not sure about  this `continue` 🤔 

one thing you can do is add a "configuration" flag, eg:
1. in the report:
```
                header += Plot_and_Text(f"...", set_config(dict(include_sleep=False), args))```
                header += Plot_and_Text(f"...", set_config(dict(include_sleep=True), args))```
```
2. in the plot:
```
cfg__instance = cfg.get("include_sleep", False)

...
if not include_sleep and "sleep" in p.pod_name: continue
```

the rational is that one way or another, I'd like to know about the scheduling of the sleep pods, as they will influence the scheduling of the build pods
"one way or another" because I'm not sure what's the best way to show them :D

Reply:
My original plan was create separate plots for each workload type. Each workload pod would get its own label and then plots for the time to complete for each workload would be added to the scheduler performance report. I'll probably try this to start.

I don't think a combined view of 2+ different workloads in one histogram would provide much information, but maybe it can show the overall performance across all workloads? Combining 2+ variables in one histogram would make it tough to draw conclusions I think

Reply:
> I don't think a combined view of 2+ different workloads in one histogram 

hum, yes, sorry, in the histogram plot it doesn't make any sense to show the `sleep` Pod 😆  🤦🏻 
```
data = generatePodTimeline(entry)
```
ok, the timeline data is generated from another function, so forget about it, it's all good !